src/s2i/bin/run

```diff
@@ -35,6 +35,13 @@ function get_default_web_concurrency() {
   echo $default
 }
 
+APP_HOME=$(readlink -f ${APP_HOME:-.})
```

Comment:
`APP_HOME` in the readlink call needs to be double-quoted to avoid word splitting